clients/java/src/main/java/io/zeebe/client/impl/oauth/OAuthCredentialsProviderBuilder.java

```diff
@@ -101,27 +108,39 @@ public OAuthCredentialsProvider build() {
   }
 
   private void checkEnvironmentOverrides() {
-    if (System.getenv("ZEEBE_CLIENT_ID") != null) {
-      this.clientId = System.getenv("ZEEBE_CLIENT_ID");
+    final String envClientId = Environment.system().get(OAUTH_ENV_CLIENT_ID);
+    final String envClientSecret = Environment.system().get(OAUTH_ENV_CLIENT_SECRET);
+    final String envAudience = Environment.system().get(OAUTH_ENV_TOKEN_AUDIENCE);
+    final String envAuthorizationUrl = Environment.system().get(OAUTH_ENV_AUTHORIZATION_SERVER);
+    final String envCachePath = Environment.system().get(OAUTH_ENV_CACHE_PATH);
+
+    if (envClientId != null) {
+      clientId = envClientId;
     }
-    if (System.getenv("ZEEBE_CLIENT_SECRET") != null) {
-      this.clientSecret = System.getenv("ZEEBE_CLIENT_SECRET");
+
+    if (envClientSecret != null) {
+      clientSecret = envClientSecret;
     }
-    if (System.getenv("ZEEBE_TOKEN_AUDIENCE") != null) {
-      this.audience = System.getenv("ZEEBE_TOKEN_AUDIENCE");
+
+    if (envAudience != null) {
+      audience = envAudience;
     }
-    if (System.getenv("ZEEBE_AUTHORIZATION_SERVER_URL") != null) {
-      this.authorizationServerUrl = System.getenv("ZEEBE_AUTHORIZATION_SERVER_URL");
+
+    if (envAuthorizationUrl != null) {
+      authorizationServerUrl = envAuthorizationUrl;
     }
-    if (System.getenv("ZEEBE_CLIENT_CONFIG_PATH") != null) {
-      this.credentialsCachePath = System.getenv("ZEEBE_CLIENT_CONFIG_PATH");
+
+    if (envCachePath != null) {
+      credentialsCachePath = envCachePath;
     }
   }
 
   private void applyDefaults() {
     if (credentialsCachePath == null) {
-      this.credentialsCachePath =
-          System.getProperty("user.home") + File.separator + ".camunda/credentials";
+      credentialsCachePath =
+          Paths.get(System.getProperty("user.home"), ".camunda", "credentials")
```

Comment:
I thought we store it under `.zeebe`?

Reply:
This is the path I use in the c# client btw

Reply:
Talked with menski, we keep it as is

Reply:
:man_shrugging: